finalPrep/modZeroHero.js

```diff
@@ -1,41 +1,55 @@
 // Challenge - See if you can follow the instructions and complete the exercise in under 30 minutes!
 
 // Declare two variables - heroName AND specialAbility - set to strings
-
+var heroName = "Speedster";
+var specialAbility = "super speed";
 // Declare two variables - greeting AND catchphrase
 //   greeting should be assigned to a string that uses concatenation to include the heroName
 //   catchphrase should be assigned to a string that uses interpolation to include the specialAbility
-
+var greeting = "Hello " + heroName;
+var catchphrase = `I'll end crime with ${specialAbility}!`
 // Declare two variables - power AND energy - set to integers
-
+var power = 709;
+var energy = 2;
 // Declare two variables - fullPower AND fullEnergy
 //   fullPower should multiply your current power by 500
 //   fullEnergy should add 150 to your current energy
-
+var fullPower = power * 500;
+var fullEnergy = energy + 150;
 // Declare two variables - isHuman and identityConcealed - assigned to booleans
-
+var isHuman = false;
+var identityConcealed = true;
 
 // Declare two variables - archEnemies AND sidekicks
 //   archEnemies should be an array of at least 3 different enemy strings
 //   sidekicks should be an array of at least 3 different sidekick strings
-
+var archEnemies = ["Slowman", "Anti-Speedster", "Speed Ticket"];
+var sidekicks = ["Zippy", "Sprinter", "Slowpoke"]
 // Print the first sidekick to your console
-
+console.log(sidekicks[0]);
 // Print the last archEnemy to the console
-
+console.log(archEnemies[2]);
```

Comment:
Another way to print the last element of the array is - `console.log(archEnemies[archEnemies.length - 1])`